Manifests/ManifestsApple/com.apple.MCXAirPort.plist

```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>pfm_app_url</key>
+	<string>https://apple.com/macos</string>	
+	<key>pfm_documentation_url</key>
+	<string>https://developer.apple.com</string>
+	<key>pfm_targets</key>
```

Comment:
Looks like in your MCXRedirector you reordered prefs to be alphabetical, which is good.  Can we do the same here, w/ a `plutil -convert xml1`.  Should automatically reorganize.